@kiva/kv-tokens/primitives.js

```diff
@@ -387,6 +387,23 @@ export default {
 					'danger-highlight': '#CE2626',
 				},
 			},
+			'eco-forest': {
```

Comment:
@emuvente Can this be removed in favor of one of the other new themes we've already added above?

Reply:
This can probably use the default theme.

Reply:
Ok cool, that makes sense to me too. This was probably one of the intermediate states before you worked with Jon to finalize and implement the themes so I'd like to avoid introducing a new theme that isn't defined in ecosystem.

Reply:
@christian14b sounds like we can remove the eco-forest theme in this PR and just use default?